docs/preview/rst/collections/index.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
```

Comment:
This file will get deleted by the rsync in `build.sh`.

Reply:
@felixfontein I see! In CI it doesn't matter, but for anyone trying to use it locally, that wouldn't be good 😅.
Since I'm using the result of `antsibull-docs init` and not actually running it each time though, removing this means it wouldn't be available for use later right?

How do you think I should proceed then? Run init as part of the process (I guess .gitignoring all the files it creates)? Should I remove the rsync step?

Reply:
Resolved based on IRC discussion. Thanks Felix!